src/main/kotlin/com/craftschemers/hub/managers/HubCommandManager.kt

```diff
@@ -0,0 +1,35 @@
+package com.craftschemers.hub.managers
+
+import com.craftschemers.hub.commands.ICommand
+import com.craftschemers.hub.commands.JoinCommand
+import com.craftschemers.hub.minigame.Minigame
+import org.bukkit.ChatColor
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+
+class HubCommandManager : CommandExecutor {
```

Comment:
Same thing as above with `JoinCommand`, unless we want to instantiate multiple instances of this, we can make this all a single `object`.

Reply:
make what a single object?